core/templates/dev/head/pages/exploration_editor/EditorServices.js

```diff
@@ -20,9 +20,10 @@
 oppia.factory('explorationData', [
   '$http', '$log', 'alertsService',
   'EditableExplorationBackendApiService',
+  'localSaveService',
   'ReadOnlyExplorationBackendApiService','$q',
   function($http, $log, alertsService, EditableExplorationBackendApiService,
```

Comment:
Let's try and have the line breaks here match those in the list of strings above, so that it's easy to compare both at a glance and verify that they're the same.